### ISSUE TITLE
rustfmt build failing on CircleCI

### ISSUE THREAD

**Issue by porcuquine** [filecoin-project/rust-fil-proofs#82]
On master a few days ago: https://circleci.com/gh/filecoin-project/rust-proofs/1424

It was working successfully on a branch until suddenly yesterday: https://circleci.com/gh/filecoin-project/rust-proofs/1472

Options:
1. Wait for it to fix itself and don't merge anything until then. (Bad option if it doesn't fix itself soon.)
2. Ignore the issue for now and merge 'failing' builds if that's the cause. (CON: Trains us to ignore red/failed builds and merged things GitHub says we shouldn't.)
3. Disable `rustfmt` checks for now. (CON: Out of sight, out of mind. We might not get back to it for awhile.)
4. Stop the world and debug it immediately. (CON: Despite being valuable and important, this is low priority compared to substantive development.)

I'm creating this discussion issue so we can actively choose which of the above we prefer. And also in case discussion helps actually resolve over time. I don't want to spend a lot of time on this right now, though.
